examples/imagenet_adv_patch.py

```diff
@@ -0,0 +1,161 @@
+"""A Shift Happens task: Model-aware/agnostic Adversarial Patch Attack on ImageNet"""
+
+import dataclasses
+import os
+
+import numpy as np
+import torch
+import torchvision.datasets as tv_datasets
+import torchvision.transforms as tv_transforms
+import torch.nn as nn
+
+import sys
+sys.path.append('./')
+
+import shifthappens.models.base as sh_models
+import shifthappens.data.base as sh_data
+import shifthappens.data.torch as sh_data_torch
+import shifthappens.utils as sh_utils
+
+from shifthappens import benchmark as sh_benchmark
+from shifthappens.data.base import DataLoader
+from shifthappens.models import base as sh_models
+from shifthappens.models.base import PredictionTargets
+from shifthappens.tasks.base import Task
+from shifthappens.tasks.base import parameter
+from shifthappens.tasks.metrics import Metric
+from shifthappens.tasks.task_result import TaskResult
+
+
+@sh_benchmark.register_task(
+    name="ImageNet_Adv_Patch", relative_data_folder="imagenet_val/", standalone=True
```

Comment:
```suggestion
    name="ImageNet_Adv_Patch", relative_data_folder="imagenet_adv_patch", standalone=True
```
We shouldn't use a path name here which is dependent on imagenet validation path.